src/components/Logs.tsx

```diff
@@ -217,22 +235,28 @@ export class Logs extends React.Component<Props> {
       const chatter = _.get(chatters, log.user.id)
       const isBanned = _.isNil(chatter) ? false : chatter.banned
       const useAlternate = alternateMessageBackgrounds && index % 2 === 0
+      const isCompressedUser = compressedUserIds.indexOf(log.user.id) >= 0
```

Comment:
Definitely not cheap (O(n)), we should use a structure where we can have direct access (maybe a POJO?)